Sources/SwiftWin32/Views and Controls/Label.swift

```diff
@@ -7,20 +7,45 @@
 
 import WinSDK
 
-public class Label: Control {
+fileprivate class Win32Label: View {
   private static let `class`: WindowClass = WindowClass(named: WC_STATIC)
+  private static let style: WindowStyle = (base: 0, extended: 0)
+
+  @_Win32WindowText
+  fileprivate var text: String?
```

Comment:
```suggestion
  public var text: String?
```

The type itself is `fileprivate` so it is okay for this to be `public`